### PR TITLE
Add solution_blocks to all levels that should now have them

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: dashboard/config/scripts/levels/2-3 Bee Conditionals 1.level [hunk 1]
```diff
@@ -39,5 +39,35 @@
...
+        <block type="when_run" movable="false">
+          <next>
+            <block type="maze_moveForward">
+              <next>
+                <block type="maze_moveForward">
+                  <next>
+                    <block type="maze_turn">
+                      <title name="DIR">turnLeft</title>
+                      <next>
+                        <block type="maze_moveForward">
```
Comment (breville) [inline dashboard/config/scripts/levels/2-3 Bee Conditionals 1.level:53]: I see maze_moveForward appears more than once.  So, solution_blocks is not just a list of the required blocks, but is actually the correct program the student should be aiming towards?

**Comment by breville** [comment]
Huge change.. looks alright to me from a cursory scan.  Do we have a good way to test this works? :)

File: dashboard/config/scripts/levels/2-3 Bee Conditionals 1.level (hunk 1 above)
Comment (Hamms) [inline dashboard/config/scripts/levels/2-3 Bee Conditionals 1.level:53]: that's correct! The idea is that this is the stuff that should be displayed when the teacher selects "View Solution" for a level

**Comment by Hamms** [comment]
Oh, I should probably mention my methodology. I wrote a simple ruby script (https://gist.github.com/Hamms/36b114c1c8b27470f568) which Asher helped me look over and which I ran on production-console to extract the current solution data for all relevant levels.

I used that data to update my local database with solutions and run some tests which verified that the solutions the system detected as failing were exactly those solutions that really are failing on prod right now.

**Comment by Hamms** [comment]
I've verified through testing that the local levels whose solutions were failing before this change are the same as the local levels whose solutions were failing after this change.

Once this goes live, the plan is to do another scan to find which levels have failing solutions so we can go in and manually update them. So if it does break things, it will break them in a way that's immediately obvious and fixable/revertable.

**Comment by breville** [comment]
Sounds good.  Is it worth committing the script for future reference?  dashboard/scripts seems to have a few, but I'm not sure what our policy is on this.  (and it should probably also get some extra comments explaining context if we do commit it)

**Comment by Hamms** [comment]
I don't think so; the long-term goal is for all future changes to the set of solutions to be communicated from levelbuilder to production and staging, so pulling solution data from production and committing it to staging and then levelbuilder like I'm doing now should never be necessary again.

**Comment by laurelfan** [comment]
I would keep the script around (and whatever scripts you use to verify that your migration worked before removing the ideal_level_sources) just so we can look at it to see what happened (not necessarily implying that we would use it again). Storage (for small text files :)) is free in github...

**Comment by Hamms** [comment]
Fair enough! I'll do so

**Comment by Hamms** [comment]
My script to validate the solutions was a change I hacked into `apps/test/levelTests/js` that definitely should not be committed.

(to validate the migration, I just ran the validation before and after the migration and confirmed that the results were identical)

One of the plans for the future of this solutions work is to automate solution validation on levelbuilder and hopefully hook it into a CD system. I'll hold on to that solution validation script locally for now, and when the times comes will rework it to fit into the new system.

Does that sound sufficient?

**Comment by laurelfan** [comment]
sure, that sounds good.

**Pull request merged**